apps/test-suite/tests/NewNotifications.js

```diff
@@ -54,5 +54,32 @@ export async function test(t) {
         t.expect(tokenFromEvent).toEqual(tokenFromMethodCall);
       });
     });
+
+    t.describe('getPermissionsAsync', () => {
+      t.it('resolves with an object', async () => {
+        const permissions = await Notifications.getPermissionsAsync();
+        t.expect(permissions).toBeDefined();
+        t.expect(typeof permissions).toBe('object');
+      });
+    });
+
+    t.describe('requestPermissionsAsync', () => {
```

Comment:
do these tests require interaction on iOS? if so, could you use something like `describeWithPermissions` here: https://github.com/expo/expo/blob/447a659538df1f87c6102919325c4c649b4f3056/apps/test-suite/tests/Calendar.js#L83

Reply:
Previously, whole `expo-notifications` has been wrapped with `describeWithPermissions`. Fixed!